packages/grafana-ui/src/components/Layout/Grid/GridItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { useStyles2 } from '../../../themes';
+
+export interface GridItemProps {
+  children: NonNullable<React.ReactNode>;
+  /** If false will the grid item inherits the display of the parent. 
+    If true, the display with 'contents' as value. See info: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents*/
```

Comment:
```suggestion
  /** If set to 'false,' the grid item will inherit the parent's display property. If set to 'true,' the display property will be set to 'contents'. More info: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents */
```
I had a hard time understanding the original comment so I tried to rephrase it a bit, but not sure if I got it right.